### ISSUE TITLE
problems in training model

### ISSUE THREAD

**Issue by tomsay** [orange90/HandWrittenChineseRecogntion#2]
I have read your code, and tried to make some experiments on HWDB1. I noticed that when I choose adam as optimizer and didn't set learn_rate myself, the train_acc fluctuated near 1/3755. On the contrary, the model kept improving when I set a learning rate scheme for it. How did you make it with neither set parameters for the optimizer nor set a learning rate?
您好，我看了您的代码并打算在HWDB1上做些实验。在直接使用adam的时候，训练模型的时候训练的acc会一直在1/3755附近波动，但是我后来自己设置lr后模型可以正常训练了。想问问您是怎么做到在不设置optimizer参数的情况下做到模型能正常训练的。谢谢。

**Comment by GodLovesJonny** [comment]
> I have read your code, and tried to make some experiments on HWDB1. I noticed that when I choose adam as optimizer and didn't set learn_rate myself, the train_acc fluctuated near 1/3755. On the contrary, the model kept improving when I set a learning rate scheme for it. How did you make it with neither set parameters for the optimizer nor set a learning rate?
> 您好，我看了您的代码并打算在HWDB1上做些实验。在直接使用adam的时候，训练模型的时候训练的acc会一直在1/3755附近波动，但是我后来自己设置lr后模型可以正常训练了。想问问您是怎么做到在不设置optimizer参数的情况下做到模型能正常训练的。谢谢。

传递优化器名的方式用默认值啊
